### ISSUE TITLE
[Bug] Unable to clone repo through contributing guide

### ISSUE THREAD

**Issue by Aju100** [hypertrons/hypertrons-crx#158]
Opensource enthusiast can't able to clone repo through [contributing guideline](https://github.com/hypertrons/hypertrons-crx#contributing)

## Process
Copy the command in the terminal
```
git clone git@github.com:hypertrons/hypertrons-crx.git

```
## Expected result
It needs to be cloned in our local machine.

## Current result

![bug](https://user-images.githubusercontent.com/29862610/120929699-c34a6d80-c709-11eb-9fda-e6f5a9457ecc.png)

## Possible Fix
Contributing guideline needs to be updated.
```
git clone https://github.com/hypertrons/hypertrons-crx/
```
From the above command, we can successfully clone the repo.

**Comment by heming6666** [comment]
Hi @Aju100 , thank you for opening this issue.

Actually, this is the difference between `HTTPS` and `SSH`. In order to clone a repository from GitHub, we can use either  `HTTPS` or `SSH`. 

Here are some reference:
- https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository
- https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/working-with-ssh-key-passphrases
- https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/error-repository-not-found

To reduce the confusion for newcomers,  you can also submit a PR to change the way from `SSH` to `HTTPS`.

**Comment by Aju100** [comment]
@heming6666 Yeah, I agreed with your views that we have to configure ssh key in our machine. But if the newcomers don't have an idea about it, they might have a problem cloning the repo and that's the reason why I raised the issue.

**Comment by Aju100** [comment]
Fixes #159
